atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraMutationTimestampProviders.java

```diff
@@ -56,8 +57,9 @@ public long getSweepSentinelWriteTimestamp() {
             }
 
             @Override
-            public long getDeletionTimestamp(long atlasDeletionTimestamp) {
-                return longSupplier.getAsLong();
+            public LongUnaryOperator getDeletionTimestampOperatorForBatchDelete() {
+                long deletionTimestamp = longSupplier.getAsLong();
+                return unused -> deletionTimestamp;
```

Comment:
Is this edit needed? Looks like it was used for debugging?

Reply:
It's needed actually - we don't want the timestamp operator to make a remote call, because it is called on every cell. Preparing the timestamp operator for a single batch delete can make a remote call (which is OK)